test/helper/healthHelper.ts

```diff
@@ -0,0 +1,63 @@
+export const responses = {
+  ok: (dscpRuntimeVersion: any, ipfsVersion: any) => ({
```

Comment:
these `any`s are presumably `string`?

Reply:
I think they are a mixture of string and number. Will update